laravel/app/Http/Controllers/InstanceController.php

```diff
@@ -25,7 +24,29 @@ class InstanceController extends Controller
      */
     public function overview(): View
     {
-        return view('instances')->with('instances', Instance::all());
+        $instances = Instance::all();
+
+        $channelListForEachInstance = [];
+        foreach ($instances as $instance) {
+            try {
+                $virtualserver_helper = new TeamSpeakVirtualserver($instance);
+                $virtualserver = $virtualserver_helper->get_virtualserver_connection();
+                $channel_list = $virtualserver->channelList();
+            } catch (ServerQueryException $serverquery_exception) {
+                return view('instance.edit', ['instance_id' => $instance->id])->with([
+                    'error' => 'instance-channellist-error',
+                    'message' => $serverquery_exception->getMessage(),
+                    'instance' => $instance,
+                    'channel_list' => [],
+                ]);
```

Comment:
[CONCERN/ERROR]

First of all: This view doesn't exist, right? So this can't work.

Also: Instead of immediately aborting the loop and returning an error, I would catch them and try the next one.

Rather have an empty list for specific instances, but show the available channels for the others, where it worked.

Also add the respective error message to this array, so that this can be properly displayed in the view in order to inform the user.

Reply:
After my changes the array and the view with channel_list or Error message works, thats the good news. 

In my test case with failure or connection Timeouts:
The view is finished loaded but the backend processes, in my option the `TeamSpeakVirtualserver($instance)`, is not finished. So i can see the blade but i cant do anything for more than 5 seconds. When the Helper function finished his Timout or Error, e.g. false Username, then i can click anywhere. 

So, i think we should testing the error handling and connection timouts intesive to understand what happens.